pages/what-we-do/partnerships/partnerships.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Partnerships
+layout: basic
+permalink: /what-we-do/partnerships/
+localnav: what-we-do
+gov-speak: true
+---
+
+Building and delivering excellent services online requires a transformation of the way services are designed and delivered by government.
+
+The DTA works in close partnership with other government agencies to challenge and change the way services are traditionally delivered, and make them better.
+
+These partnerships range in scale and duration—from providing short-term specialist skills, support and training to long-term intensive partnerships from which major new online services are developed and delivered.
```

Comment:
Content guide says to use spaced em-dashes: http://content-guide.apps.staging.digital.gov.au/punctuation-grammar/#em-dashes-mdash